src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -747,6 +751,7 @@ object ManagedGroupRoutesSpec {
   ): TestSamRoutes = {
     val directoryDAO = new MockDirectoryDAO()
     val policyDao = new MockAccessPolicyDAO(resourceTypeMap, directoryDAO)
+    implicit val openTelemetry: OpenTelemetryMetricsInterpreter[IO] = mock[OpenTelemetryMetricsInterpreter[IO]](RETURNS_SMART_NULLS)
```

Comment:
could've used https://github.com/broadinstitute/workbench-libs/blob/develop/openTelemetry/src/test/scala/org/broadinstitute/dsde/workbench/openTelemetry/FakeOpenTelemetryMetricsInterpreter.scala#L8 so that u don't have to mock everywhere

Reply:
I did see that but when I tried to drop that in, the types were mismatched. So then I went with mockito because it was simple to mock out and reduces the reliance on workbench-libs. I could see both approaches working though! Maybe we'll switch to using `FakeOpenTelemetryMetricsInterpreter` as we add more metrics!

Reply:
aha! that makes sense. thanks for the info!